src/lrf.rs

```diff
@@ -139,13 +139,7 @@ impl Default for RestorationFilter {
 impl RestorationFilter {
   pub fn notequal(self, cmp: RestorationFilter) -> bool {
     match self {
-      RestorationFilter::None {} => {
-        if let RestorationFilter::None {} = cmp {
-          false
-        } else {
-          true
-        }
-      }
+      RestorationFilter::None {} => matches!(cmp, RestorationFilter::None {}),
```

Comment:
@barrbrain pointed out there is a `!` missing here

Reply:
Updated. Sorry about that and thanks for catching it.